projects/demo/src/modules/components/combo-box/examples/6/index.ts

```diff
@@ -0,0 +1,47 @@
+import {Component} from '@angular/core';
+import {FormControl} from '@angular/forms';
+import {changeDetection} from '@demo/emulate/change-detection';
+import {encapsulation} from '@demo/emulate/encapsulation';
+import {TuiStringHandler} from '@taiga-ui/cdk';
+import {
+    tuiComboBoxOptionsProvider,
+    tuiDataListWrapperOptionsProvider,
+} from '@taiga-ui/kit';
+
+interface Employee {
+    readonly id: number;
+    readonly name: string;
+    readonly dept: {
+        readonly id: number;
+        readonly title: string;
+    };
+}
+
+const STRINGIFY_EMPLOYEE_SELECT: TuiStringHandler<Employee> = (item: Employee) =>
+    `${item.name} (${item.dept.title})`;
+
+const STRINGIFY_EMPLOYEE_LIST: TuiStringHandler<Employee> = (item: Employee) =>
```

Comment:
These are the same it seems :)

Reply:
Not quite, when an item is selected, department title is in parentheses – this way items in multi-select will not look like a pile of junk.